src/controller.cpp

```diff
@@ -361,6 +487,14 @@ void controller::run(int argc, char * argv[]) {
 
 	LOG(level::INFO, "nl_langinfo(CODESET): %s", nl_langinfo(CODESET));
 
+	if ((! using_nonstandard_configs) && (0 != access(url_file.c_str(), R_OK))) {
+		migrate_data_from_newsbeuter(silent);
+	} else {
+		utils::mkdir_parents(config_dir.c_str(), 0700);
```

Comment:
If newsbeuter migration is attempted (but doesn't happen) this part of the code is never reached meaning that the directories aren't created and the cache lock fails.

Reply:
Oops, forgot to address this one! Stay tuned for another commit...